.github/workflows/docker-build.yml

```diff
@@ -5,10 +5,9 @@ on:
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}/dlio_benchmark
 
 jobs:
-  build-and-push:
+  build-dlio-benchmark:
```

Comment:
add an option to specific what targets should be built (comma separated). By default build everything.